content.js

```diff
@@ -1,54 +1,75 @@
 // variables
 let recording = false;
-let count=0, maxCount=0;
+let count = 0,
+  maxCount = 0;
+let minValue = null,
```

Comment:
While I understand it's not wrong, I'd like either each of the variables have separate declarations on separate lines, or them being in a single line, sandwiched between the data type ('let' here) and the semi-colon

let count=0, maxCount=0;
let minValue=null, maxValue=null;

Would be best